src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -175,31 +185,46 @@ const ScenarioParameters = ({
 
   // Normal Mode Screen
   const handleClickOnEditButton = () => changeEditMode(true);
-  const isCurrentScenarioRunning = () => (
-    currentScenario.data.state === SCENARIO_RUN_STATE.RUNNING);
+  const isCurrentScenarioRunning = () => (currentScenario.data.state === SCENARIO_RUN_STATE.RUNNING);
 
   const handleClickOnLaunchScenarioButton = () => {
-    // If scenario parameters have never been updated, do it now
-    if (!currentScenario.data.parametersValues) {
-      handleClickOnUpdateAndLaunchScenarioButton();
-    } else {
-      launchScenario(workspaceId, scenarioId);
-      changeEditMode(false);
-    }
+    launchScenario(workspaceId, scenarioId);
+    changeEditMode(false);
```

Comment:
This change may break the current behavior when clicking on "launch" just after a scenario creation, unless the parametersValues have been set by the back-end. Was it done on purpose ?

Reply:
:+1: